### PR TITLE
Fix issue #103

### PR DESCRIPTION
**Pull request by kalletlak**

### CODE REVIEW

File: app/scripts/vueCore.js [hunk 1]
```diff
@@ -131,6 +133,114 @@
              };
              this.charts = {};
              },*/
+            openCases:function(){
+              iViz.openCases();
+              
+             /* var studyId = '';
```
Comment (zhx828) [inline app/scripts/vueCore.js]: Can we remove these commented out code? I think they have been isolated into an individual function?

File: app/scripts/vueCore.js [hunk 2]
```diff
@@ -262,6 +372,21 @@
...
             });
+            $('.study-view-header-left-5').qtip({
+              content: {text: 'Click to view the selected cases' },
+              style: { classes: 'qtip-light qtip-rounded qtip-shadow' },
+              show: {event: 'mouseover'},
+              hide: {fixed:true, delay: 100, event: 'mouseout'},
+              position: {my:'bottom center', at:'top center', viewport: $(window)}
+            });
+            $('#study-view-header-left-6').qtip({
+              prerender: true,
```
Comment (zhx828) [inline app/scripts/vueCore.js]: Why prerender the qtip?

File: app/styles/partials/_global.scss [hunk 3]
```diff
@@ -72,14 +72,14 @@
 }
 
 #study-view-header-right-1 {
```
Comment (zhx828) [inline app/styles/partials/_global.scss]: Should we get rid of 'study-view' name?

**Comment by kalletlak** [comment]
Created another PR #147 to accommodate same changes.

**Pull request closed without merging**